src/codegen.cpp

```diff
@@ -357,6 +357,7 @@ static Function *jlboundserrorv_func;
 static Function *jlcheckassign_func;
 static Function *jldeclareconst_func;
 static Function *jlgetbindingorerror_func;
+static Function *jlboundp_func;
```

Comment:
What's the conclusion, do I need to add renaming of this function to the PR, or is this gtg?